### ISSUE TITLE
Windows + Pinokio installs CPU-only PyTorch instead of CUDA version

### ISSUE THREAD

**Issue by Bacrima** [pinokiocomputer/pinokio#1105]
Environment
Windows 11
NVIDIA GeForce RTX 3080
NVIDIA Driver 591.74
CUDA Driver 13.1
Fresh installation using Pinokio
Python 3.10 (Conda environment created by Pinokio)
Problem

After a fresh installation with Pinokio, Seed-VC was running extremely slowly.

Processing 30 seconds of audio took between 6 and 10 minutes, and GPU usage stayed around 0–2%.

nvidia-smi showed almost no compute activity.

Running:

python -c "import torch; print(torch.__version__); print(torch.version.cuda); print(torch.cuda.is_available())"

returned:

2.4.0+cpu
None
False

So the environment had installed the CPU build of PyTorch.

Strange part

The installation scripts appear to request the CUDA build:

uv pip install torch==2.4.0 torchvision==0.19.0 torchaudio==2.4.0 --index-url https://download.pytorch.org/whl/cu124

Yet the resulting installation was still:

torch 2.4.0+cpu
Manual fix

I manually reinstalled the CUDA build:

python -m pip install --force-reinstall torch==2.4.0 torchvision==0.19.0 torchaudio==2.4.0 --index-url https://download.pytorch.org/whl/cu124

After that:

python -c "import torch; print(torch.__version__); print(torch.version.cuda); print(torch.cuda.is_available())"

returned:

2.4.0+cu124
12.4
True

This introduced two dependency issues that also had to be fixed:

python -m pip install numpy==1.26.4
python -m pip install "typing_extensions>=4.12"
Result

After these changes:

GPU usage jumped to ~95%
Processing time dropped from 6–10 minutes per 30-second chunk to about 3 seconds
Roughly a 100×–200× speedup

This suggests the installation may silently install the CPU version of PyTorch on Windows even though the installer requests the CUDA wheel.

It might be worth checking whether uv is resolving the correct package from the CUDA index during installation.
